project/plugins.sbt

```diff
@@ -2,6 +2,6 @@ resolvers += Classpaths.sbtPluginReleases
 resolvers += Resolver.sonatypeRepo("snapshots")
 
 addSbtPlugin("com.typesafe.sbt" % "sbt-site" % "1.4.1")
-addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.9.1")
+addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.9.3")
```

Comment:
sbt-scoverage 2.0.0 is now out and upgrade seems to be smooth (from my experience with other projects)

Reply:
I tried that first but ran into all sorts of problems, so I changed the PR to be more conservative. My main goal here is using a recent Scala 3 version.

I only update Scala 2.x versions until it became painful to go further (e.g. 2.12.16 and 2.13.8 both caused issues).